### PR TITLE
feat!: migrate `maplibre-native` for android to version `11.0.0` (take 2)

### PR DESCRIPTION
**Pull request by josxha**
This pull request updates the package to use version 11.0.0 of maplibre-native for android. It superseeds https://github.com/maplibre/flutter-maplibre-gl/pull/403 to make it not dependant on other pull requests.

If wanted, we can change the occurences of "Maplibre" to "MapLibre" to be more in line with maplibre-native. (I'd prefer to do it in a follow-up pr tho).

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -46,7 +46,7 @@ android {
         jvmTarget = JavaVersion.VERSION_1_8
     }
     dependencies {
-        implementation 'org.maplibre.gl:android-sdk:10.2.0'
+        implementation 'org.maplibre.gl:android-sdk:11.0.0'
         implementation 'org.maplibre.gl:android-plugin-annotation-v9:2.0.0'
         implementation 'org.maplibre.gl:android-plugin-offline-v9:2.0.0'
```
Comment (kuhnroyal) [inline android/build.gradle]: You need to bump these to 3.0.0 :)

Comment (josxha) [reply android/build.gradle]: 🤯Thanks a lot for looking into this!

**Comment by josxha** [comment]
Fixed the version of the MapLibre plugins. 
There are newer versions for [androidx.core:core-ktx](https://mvnrepository.com/artifact/androidx.core/core-ktx) too. Should we update it or is it better to keep it on an older version?

**Comment by kuhnroyal** [comment]
We can do that separately.

**Comment by kuhnroyal** [comment]
Sadly I only get black views instead of maps, something is wrong, not sure what.

**Comment by josxha** [comment]
That's strange. It works fine for me on an physical device with android 11.
What android version were you testing on?

**Comment by kuhnroyal** [comment]
Only simulator so far, but the main branch works on simulator.

**Comment by josxha** [comment]
Could you check what android or sdk version it's using? Maybe it's related or similar to https://github.com/maplibre/flutter-maplibre-gl/issues/327 which depends on the used OS version.

**Comment by kuhnroyal** [comment]
Dunno why but it is working now :)

**Comment by kuhnroyal** [comment]
And also works on device.
I have not reviewed code yet, only tested.

**Comment by josxha** [comment]
> Dunno why but it is working now :)

Awesome!👍

**Review by kuhnroyal — APPROVED** [review]
Looks pretty straight forward, nothing to add :)

**Comment by josxha** [comment]
Thanks again for the help on this pr! (:

**Pull request merged**